thirdparty/openapi3/schemas/ConsentRevokedAt.yaml

```diff
@@ -0,0 +1,5 @@
+title: ConsentRevokedAt
```

Comment:
Thoughts on removing this and just using https://github.com/mojaloop/api-snippets/blob/master/v1.0/openapi3/schemas/DateTime.yaml .

i.e for `Consent`
```
  revokedAt:
    $ref: '../../../v1.0/openapi3/schemas/Datetime.yaml'
```